### PR TITLE
addAdmin solana cli admin cmd

### PR DESCRIPTION
**Pull request by zurgl**
Allow to add new Admin to the kafe list using builderdao cli

### CODE REVIEW

**Review by Necmttn — APPROVED** [review]
LGTM, I also think there should be quick CLI command to remove admin due to it's possible to add the wrong admin by mistake (somehow).

File: packages/cli/src/commands/admin/removeAdmin.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import ora from 'ora';
+
+import { getClient } from '../../client';
+import { createKeypairFromSecretKey } from '../../utils';
+
+const helpText = `
+Add an admin Solana pubkey to the Kafe Admin list
+
+Example call to admin:
+$ builderdao admin addAdmin --adminKp <bs58Secret> --address <bs58Pubkey>
```
Comment (pyr0gan) [inline packages/cli/src/commands/admin/removeAdmin.ts]: Should be 

```text
$ builderdao admin removeAdmin --adminKp <bs58Secret> --address <bs58Pubkey>
```

File: packages/cli/src/commands/admin/removeAdmin.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+`;
+
+export const AdminRemoveAdmin = () => {
+  const removeAdmin = new commander.Command('addAdmin')
```
Comment (pyr0gan) [inline packages/cli/src/commands/admin/removeAdmin.ts]: Needs to be `removeAdmin` 😁

File: packages/cli/src/commands/admin/removeAdmin.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    .description('Add Admin to Kafe List')
```
Comment (pyr0gan) [inline packages/cli/src/commands/admin/removeAdmin.ts]: > 'Remove Admin from Kafe List'

File: packages/cli/src/commands/admin/removeAdmin.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+/* eslint-disable no-restricted-syntax */
+import * as commander from 'commander';
+import * as anchor from '@project-serum/anchor';
+import ora from 'ora';
+
+import { getClient } from '../../client';
+import { createKeypairFromSecretKey } from '../../utils';
+
+const helpText = `
+Add an admin Solana pubkey to the Kafe Admin list
```
Comment (pyr0gan) [inline packages/cli/src/commands/admin/removeAdmin.ts]: > Remove an admin Solana pubkey from the Kafe Admin list

**Review by pyr0gan — COMMENTED** [review]
Minor text edits needed.

**Review by Necmttn — APPROVED** [review]
LGTM

**Review by pyr0gan — APPROVED** [review]

**Pull request merged**